packages/cleanup-bm.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -euo pipefail
+
+echo "Starting cleanup"
+
+mapfile -t runtimeClasses < <(kubectl get runtimeclass -o jsonpath='{.items[*].metadata.name}' | tr ' ' '\n' | grep '^contrast-cc')
+
+declare -a usedRuntimeClasses=()
+declare -a unusedRuntimeClasses=()
```

Comment:
I find the logic in this script hard to follow. Have you considered just filling a temp file with runtimeclasses in use, and using `comm(1)` to do the set difference?